vm/src/builtins.rs

```diff
@@ -430,8 +430,17 @@ mod decl {
 
     #[pyfunction]
     fn max(vm: &VirtualMachine, args: PyFuncArgs) -> PyResult {
+        let default = args.get_optional_kwarg("default");
```

Comment:
How about using `take_keyword` here? Then we can check invalid keyword just by testing it is empty instead of loop.